database/boltdb.go

```diff
@@ -0,0 +1,135 @@
+package database
+
+import (
+	"bytes"
+	"errors"
+	bolt "github.com/coreos/bbolt"
```

Comment:
please use `goimports` to format your code.

https://golang.org/x/tools/cmd/goimports

Reply:
Sure. Actually entire formatting is yet to be done.